src/util.cc

```diff
@@ -221,6 +221,39 @@ int WriteFileSync(v8::Isolate* isolate,
   return WriteFileSync(path, buf);
 }
 
+int ReadFileSync(std::string* result, const char* path) {
+  uv_fs_t req;
+  uv_file file = uv_fs_open(nullptr, &req, path, O_RDONLY, 0, nullptr);
+  if (req.result < 0) {
+    return req.result;
+  }
+  uv_fs_req_cleanup(&req);
```

Comment:
Elsewhere we call `uv_fs_req_cleanup` regardless of the value of `req.result`, maybe we should do the same here?

Reply:
Fixed in https://github.com/nodejs/node/pull/38904